### ISSUE TITLE
Currency in dashboard

### ISSUE THREAD

**Issue by RolfMoleman** [infracost/infracost#2829]
The currency symbol does not change in the infracost cloud dashboard, other than for specific pull requests where the curency has been predefined.

it would be nice to be able to change the currency in user settings for a user specific currency, and also to be able to set an organisation currency.

**Comment by alikhajeh1** [comment]
@RolfMoleman the currency showing inconsistently in the Infracost Cloud dashboard is indeed a bug.

Interesting idea about each user being able to adjust their own currency, how do you see that working with PRs? I guess the organization's currency is used in all PRs? So the user setting only applies to Infracost Cloud.

**Comment by RolfMoleman** [comment]
@alikhajeh1 precisely. Unless the currency is overridden in the pr pipeline via the cli commands

**Comment by glenngillen** [comment]
We're moving to a contribution model where Issues will be exclusively for work that is ready to be worked on. Discussions and feature requests about potential future capabilities will be over in [Discussions](https://github.com/infracost/infracost/discussions). There has been very little community or customer interest in this capability for an extended period, but if it still interests you (or you're now reading this issue because you've been looking for a solution) please [create a new thread in Discussions](https://github.com/infracost/infracost/discussions).
